commands/ping.go

```diff
@@ -17,41 +19,65 @@ var (
 		Use:   "ping",
 		Short: "Check API connection",
 		Long:  `Tests the connection to the API`,
-		Run:   ping,
+		Run:   runPingCommand,
 	}
 )
 
 func init() {
 	RootCommand.AddCommand(PingCommand)
 }
 
-// ping checks the API connections
-func ping(cmd *cobra.Command, args []string) {
-	u, err := url.Parse(cmdAPIEndpoint)
+// runPingCommand executes the ping() function
+// and prints output in a user-friendly way
+func runPingCommand(cmd *cobra.Command, args []string) {
+	duration, err := ping(cmdAPIEndpoint)
 	if err != nil {
-		fmt.Println(err.Error())
+		fmt.Println(color.RedString("API connection is bad"))
```

Comment:
I don't really like "API connection is bad" from a UX point of view.

Can we offer some more information here? Or at least something more concrete like "Could not reach API" ? It's so very subjective to hear something is bad.. I am left wondering what it means.